apps/token-manager/contracts/TokenManager.sol

```diff
@@ -301,7 +300,7 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
     }
 
     function spendableBalanceOf(address _holder) public view isInitialized returns (uint256) {
-        return _transferableBalance(_holder, now);
+        return _spendableBalanceOf(_holder);
     }
```

Comment:
If this is public (not external) why do we need the wrapper and the internal function here? Can't we just use one as before?

Reply:
This is used in `onTransfer()` as well, so I split it up into the public / internal functions.

Reply:
So the goal is to save checking `isInitialized`? Couldn't we just do `_transferableBalance(_holder, now)` from there? It seems like unnecessary complexity to me.

Reply:
It is a little bit more expensive with the extra jump, but I was thinking it'd be a lot more clear to reuse the functionality.

However, since this is getting used on every token transfer, it's probably worth it to be as optimized as possible 👍.